packages/fast-components-styles-msft/src/utilities/high-contrast.ts

```diff
@@ -8,45 +8,75 @@ export const highContrastSelector: string = "@media (-ms-high-contrast:active)";
 
 export enum HighContrastColor {
     text = "WindowText",
-    hyperLinks = "LinkText",
+    forcedColorLink = "LinkText",
+    msLink = "-ms-hotlight",
     disabledText = "GrayText",
     selectedText = "HighlightText",
     selectedBackground = "Highlight",
     buttonText = "ButtonText",
     buttonBackground = "ButtonFace",
-    background = "Background",
+    background = "Window",
 }
 
-// Used to remove text backplate and borders in 'button-text' colors
+// Function used to to opt-out of high contrast color scheme for '-ms' prefixes
+export function applyhighContrastOptOutProperty(): CSSRules<{}> {
+    return {
+        "-ms-high-contrast-adjust": "none",
+    };
+}
+// Used to to opt-out of high contrast color scheme
+export const highContrastOptOutProperty: CSSRules<{}> = applyhighContrastOptOutProperty();
+
+// Function used to to set link color base on 'forced-color' query
+export function applyHighContrastlinkValue(): string {
+    return window.matchMedia("(forced-colors: none)").matches ||
```

Comment:
I think you should break this out further - can you create a `supportsForcedColors` in `fast-web-utilities` and use that function here?

Reply:
add function to the dom file in fast-web-utilities